app/models/manageiq/providers/cloud_manager/orchestration_stack.rb

```diff
@@ -19,6 +19,10 @@ def total_vms
     vms.size
   end
 
+  def indirect_vms
```

Comment:
Looks like this is a great choice once we convert over to closure_trees. Will look at this when we make that conversion.

Holding off for now